torch/_inductor/utils.py

```diff
@@ -992,9 +992,7 @@ def is_big_gpu(index) -> bool:
 
 
 def use_max_autotune() -> bool:
-    return (
-        config.max_autotune or config.max_autotune_gemm or config.search_autotune_cache
-    )
+    return config.max_autotune or config.max_autotune_gemm
```

Comment:
making config.search_autotune_cache a no-op, in the current state this is totally unused internally due to various concerns and real issues (autotuning and caching are already hard enough on their own, and I didn't do enough to get them working together seamlessly).